src/GUI/UI/UIHouses.cpp

```diff
@@ -440,18 +427,23 @@ void PrepareHouse(HOUSE_ID house) {
         }
     }
 
-    for (int i = 0; i < npcs; ++i) {
-        pDialogueNPCPortraits.push_back(assets->getImage_ColorKey(fmt::format("npc{:03}", npc_id_arr[i])));
-    }
+    // Dungeon entry (not present in MM7)
```

Comment:
We've discussed that it actually is present, no?

Reply:
Nope.
This "entry" opens special transition window and uses UIMSG_BF for confirmation. Which is unused in MM7 - at least I dumped building table and searched for corresponding IDs, they all zero.
What we have in MM7 (I checked Pit temple) is regular NPC called "temple". And you have regular conversation with it to enter the dungeon.